js-envs/src/main/scala/org/scalajs/jsenv/phantomjs/PhantomJSEnv.scala

```diff
@@ -493,13 +488,14 @@ class PhantomJSEnv(
     }
 
     protected def writeCodeLauncher(code: VirtualJSFile, out: Writer): Unit = {
-      out.write("""<script type="text/javascript">""" + "\n")
-      out.write("// Phantom.js code launcher\n")
-      out.write(s"// Origin: ${code.path}\n")
-      out.write("window.addEventListener('load', function() {\n")
-      out.write(code.content)
-      out.write("}, false);\n")
-      out.write("</script>\n")
+      // Create a file with the launcher function.
+      val launcherFile = new MemVirtualJSFile("phantomjs_launcher.js")
```

Comment:
Consider using `-` instead of `_` so that we keep being uniform about this across the codebase.